src/XMakeCommandLine/Resources/xlf/Strings.cs.xlf

```diff
@@ -327,7 +327,7 @@ Copyright (C) Microsoft Corporation. Všechna práva vyhrazena.
         <target state="translated">  /consoleloggerparameters:&lt;parametry&gt;
                      Parametry protokolovacího nástroje konzoly. (Short form: /clp)
                      Dostupné parametry:
-                        PerformanceSummary – zobrazí dobu zpracování úloh,
+                        PerformanceSummary –  zobrazí dobu zpracování úloh,
```

Comment:
Are the whitespace-only changes in this file intentional? The old spacings look OK to me (but I don't know the language).